Doc/whatsnew/3.12.rst

```diff
@@ -1713,10 +1713,10 @@ Build Changes
   if the Clang compiler accepts the flag.
   (Contributed by Dong-hee Na in :gh:`89536`.)
 
-* Add ``COMPILEALL_OPTS`` variable in Makefile to override :mod:`compileall`
-  options (default: ``-j0``) in ``make install``. Also merged the 3
-  ``compileall`` commands into a single command to build .pyc files for all
-  optimization levels (0, 1, 2) at once.
+* Add ``COMPILEALL_OPTS`` variable in the :file:`Makefile` to override
+  :mod:`compileall` options (default: ``-j0``) in ``make install``.
+  Also merged the 3 ``compileall`` commands into a single command to
+  build .pyc files for all optimization levels (0, 1, 2) at once.
```

Comment:
Let's avoid re-wrapping text.

It's okay for revisions to extend beyond the line limit, and it makes it much easier to review.

Reply:
My error -- I assumed that the sphinx-lint's line-limit check was enabled without verifying this, and I preemptively wrapped to avoid the lint failure!